index.html

```diff
@@ -2,12 +2,22 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+      <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
   </head>
   <body>
     <div id="app"></div>
+    <button id="hoverTooltipButton">Tooltip on hover</button>
```

Comment:
Don't know what to say, all clean and well done
Just one more note:
It is good habit to have when you crate a button to add `type="button"` as parameter.
The fact is that all buttons has default behaviour as "submit" and if you place your interface to <form> tag all buttons will do more than you really need.